packages/ember-data/lib/system/store.js

```diff
@@ -1071,13 +1072,14 @@ Store = Service.extend({
     once the server returns.
 
     @method queryRecord
-    @param {String or subclass of DS.Model} type
-    @param {any} query an opaque query to be used by the adapter
+    @param {String} modelName
+    @param {Hash} options, query an opaque query of the form {query: { any }} to be used by the adapter
```

Comment:
The API docs prefer the term `Object` instead of `Hash` for the type annotations. Same for line 1045 above.